GSA/usagov-benefits-eligibility
# 760 update tab order

## PR Summary

update tab order to match parent site, usa.gov

## Related Github Issue

- #760 

## Type of change

Please delete options that are not relevant.


## Branch Name

760-update-tab-order

## Testing environment

Link to a Federalist URL or to be tested on local dev environment.

## Detailed Testing steps

navigate to feature branch testing environment, start tabbing into the site elements

- [ ] Tabbing order should go as follows

1. Skip to main content
2. banner
3. language switcher
4. logo
5. phone number
6. search input
7. search submit
8. main nav

etc...

## Checklist:
